src/application/index.ts

```diff
@@ -67,13 +65,30 @@ class JupyterLab extends Application<ApplicationShell> {
   }
 
   /**
+   * Promise that resolves when state is restored, returning layout description.
+   *
+   * #### Notes
+   * This is just a reference to `shell.restored`.
```

Comment:
That's a good point. But I didn't want people to be confused because `app.shell.restored` is *also* publicly accessible, so I thought making it clear that they are identical was prudent. What do you think?